mainnet-production.json

```diff
@@ -232,6 +238,35 @@
   },
   {
     "env": "production",
+    "version": 3,
+    "startblock": 1,
+    "addresses": {
+      "ROOT_CONTRACT": "0x0",
+      "TINLAKE_CURRENCY": "0x0",
+      "TITLE": "0x0",
+      "PILE": "0x0",
+      "SHELF": "0x0",
+      "COLLECTOR": "0x0",
+      "FEED": "0x0",
+      "JUNIOR_OPERATOR": "0x0",
+      "SENIOR_OPERATOR": "0x0",
+      "JUNIOR_TRANCHE": "0x0",
+      "SENIOR_TRANCHE": "0x0",
+      "JUNIOR_TOKEN": "0x0",
+      "SENIOR_TOKEN": "0x0",
+      "JUNIOR_MEMBERLIST": "0x0",
+      "SENIOR_MEMBERLIST": "0x0",
+      "ASSESSOR": "0x0",
+      "COORDINATOR": "0x0",
+      "RESERVE": "0x0",
+      "GOVERNANCE": "0x0",
+      "ACTIONS": "0x0",
+      "PROXY_REGISTRY": "0x0"
+    },
+    "contractConfig": {
+      "JUNIOR_OPERATOR": "ALLOWANCE_OPERATOR",
+      "SENIOR_OPERATOR": "PROPORTIONAL_OPERATOR"
+    },
```

Comment:
These are not used for v3 anymore but should be left in the pool config for now.